core/src/test/scala/com/rallyhealth/vapors/factfilter/evaluator/FilterOutputSpec.scala

```diff
@@ -12,141 +12,136 @@ class FilterOutputSpec extends AnyWordSpec {
 
   "Expr.FilterOutput" when {
 
-    "using containsAny op" should {
+    "using with an 'OutputWithinSet' op" when {
```

Comment:
It might be easier to review this file by just looking at the whole side-by-side. I rearranged the tests so that everything lives under this `"using with an 'OutputWithinSet' op"` so that I have space to add tests for passing different condition expressions than just the `OutputWithinSet` operator -- namely, the `OutputWithinRange` operator.